articles/osquery-evented-tables-overview.md

```diff
@@ -0,0 +1,162 @@
+- [What do I need to consider when configuring evented tables?](#what-do-i-need-to-consider-when-configuring-evented-tables?)
+- [How do I turn on an evented table?](#how-do-i-turn-on-an-evented-table?)
+-- [File integrity monitoring](#file-integrity-monitoring-fim)
+-- [Process auditing](#process-auditing)
+-- [Yara scanning](#yara-scanning)
+
+# Osquery evented tables
+
+## What are osquery evented tables? 
+Osquery traditionally allows you to see the state of your computers right now. This point-in-time snapshot, however, requires you to actively watch the table repeatedly. For example, in the security-critical `processes` table, a process might be very short-lived and there is high churn in processes. Actively watching and diffing this table would be challenging even with automation.
+
+This is where osquery's evented tables come in. Rather than displaying the point-in-time state for your host, osquery's evented tables store the host's historical data. You can configure osquery to capture certain types of information, after which it will be stored in the relevant `*_events` table for later analysis. 
+
+This guide will walk you through the concepts, considerations, and best practices for setting up evented tables, in particular focusing on osquery. It will also provide basic information about commonly-used evented tables for a newcomer to get started.
+
+## How do osquery evented tables work?
+Osquery does not generate the events itself. Instead, it reads and formats event data generated by system utilities. For example, on Linux, audit generates process and socket event data in its own storage location and format. Osquery reads data from audit’s storage location, converts it into an event row, and buffers them (in the internal [rocksdb](http://rocksdb.org) store) in the `process_events` and `socket_events` tables to await querying. The data can then be filtered and transformed via SQL, and shipped to a log destination with the scheduled query functionality. 
+
+For the purposes of this article, we will use the term "utility" to mean the underlying OS utility that osquery subscribes to for its various evented tables. 
+
+This separation between osquery and the utility means that some evented tables rely on configurations for the utility to determine which events will be generated. At the utility level, you can specify what data is captured and at the osquery level, you can specify what information is ingested, presented, and transmitted. For most evented tables, osquery works great out-of-the-box with the utility’s default configurations,but some use cases may require adjusting the utility configuration.
+
+## How do I know whether a table is evented?
+You can tell that an osquery evented table is evented in two ways:
+
+The table ends in `_events`.
+In the (osquery schema)[osquery.io/schema], the table is  marked with an "evented" tag near the table name as shown here:
+
+IMAGE NEEDED HERE
```

Comment:
@chris-mcgillicuddy I'm not sure how to add images into the article... I was thinking something like this?

![image](https://user-images.githubusercontent.com/5359586/189983213-f614dafc-0dbc-4772-915f-1fa4de1bc424.png)

Reply:
@zhumo https://fleetdm.com/handbook/digital-experience/article-formatting-guide#images-and-screenshots